app/assets/stylesheets/content/_timelines.scss

```diff
@@ -71,6 +71,11 @@
   }
 }
 
+/*HACK: Use a min-height to ensure that the timeline_select select2 is not cropped at the bottom*/
```

Comment:
@NobodysNightmare better tooling (e.g. code-linting) is compelling reason for moving to SCSS syntax everywhere: https://community.openproject.org/work_packages/18422